doc/api.rst

```diff
@@ -51,6 +51,19 @@ Classification
     FgMDM
     TSclassifier
     KNearestNeighbor
+    SVC
+
+Regression
+--------------
+.. _regression_api:
+.. currentmodule:: pyriemann.regression
+
+.. autosummary::
+    :toctree: generated/
+    :template: class.rst
+
+    SVR
+    KNNRegression
```

Comment:
The name of the regressor (currently, KNNRegression) should be homogeneous with the classifier one (KNearestNeighbor). 

Suggestion 1 (requiring a deprecation):
KNearestNeighbor -> KNNClassifier
KNNRegression -> KNNRegressor

Suggestion 2:
KNearestNeighbor
KNNRegression -> KNearestNeighborRegressor

Suggestion 3 (use sklearn names, requiring a deprecation):
KNearestNeighbor -> KNeighborsClassifier
KNNRegression -> KNeighborsRegressor

Reply:
I opted for 2) as it does't require deprecation.